services/QuillLMS/client/app/bundles/Tools/startup/subNavbarAppClient.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { SubNavbar } from '../../Shared';
+
+const SubNavbarApp = (props) => (
```

Comment:
Is the reason for the inconsistency between the name of the file `subNavbarAppClient` and the const name `subNavbarApp` a historical, conventional, required?  (It kind of breaks the symmetry with naming in other files.)

Reply:
So if you look at newer files, @emilia-friedberg and I have been following a naming convention of PascalCase for component naming and camelCase for files.

Official React docs are unopinionated, but the general consensus is that component naming should be PascalCase, whereas file naming is more ambiguous: https://stackoverflow.com/questions/43973199/file-naming-conventions-in-reactjs

I think we could maybe have a larger discussion about this an engineering meeting though since this isn't explicitly documented anywhere.

Reply:
My question was not about the camelcasing but that the const is called `SubNavbarApp` and the file is named `subNavbarAppClient` (i.e. client is used in one but not the other).

Reply:
Oh, right-- that's just a historical convention. We could probably do some cleanup there during the summer months to make all the naming conventions consistent